transformerlab/routers/data.py

```diff
@@ -59,28 +59,24 @@ async def dataset_info(dataset_id: str):
 
 
 @router.get("/preview", summary="Preview the contents of a dataset.")
-async def dataset_preview(dataset_id: str):
+async def dataset_preview(dataset_id: str, offset: int = 0, limit: int = 10):
```

Comment:
Please document the function so that it appears in proper documentation format in the API spec